engineering/workflow.md

```diff
@@ -54,31 +55,153 @@ The team conducts a Sprint Planning meeting for 60 minutes at the beginning of e
 The goal of this meeting is to review our major work items, synchronize with one another, and prioritize work across team members.
 
 See [the Sprint Planning issue template](https://github.com/2i2c-org/team-compass/blob/main/.github/ISSUE_TEMPLATE/meeting-sprint-planning.md) for the agenda / structure of these meetings.
- 
+
 (coordination:sprint-board)=
 ### The Sprint Board
 
-The Sprint Board is a place to keep track of the [Deliverables and tasks](coordination:deliverables) our team intends to work on for the week.
-It is a GitHub Projects board that is created for each week, and closed at the end of each week.
+The [Sprint Board](https://github.com/orgs/2i2c-org/projects/49/views/1) is a place to keep track of the [Deliverables and tasks](coordination:deliverables) our team intends to work on for a two week iteration.
+
+The board is a GitHub Projects board that is populated with with work items during the teams Itereation Planning activity.
+
 The team's goal is to complete all items on the Sprint Board by the end of the Sprint.
 This is a team commitment - while one person may be assigned to a deliverable, we all commit to working together to get the work done.
 
 :::{admonition} The Sprint Board should...
 :class: tip
-- Have enough items to keep the team occupied for the week
+- Be driven by 2i2c's product roadmap
+- With the exception of support tickets, work items should be related back to Initiatives.
+- Have enough items to keep the team occupied for the two week iteration
 - Not have so many items that a team member gets overwhelmed
 - Under-estimate our team's total capacity, to provide room for unexpected work (e.g., support work)
-- Have a team member assigned to each item on the board
+- Have at least one team member assigned to each item on the board
 :::
 
-The Sprint Board is broken down into these columns:
+The Sprint Board is broken down into different columns that represent the teams delivery workflow. The team owns the design of this workflow and should change the workflow process that best suits their way of working to optimize for sustainable delivery.
+
+The current queues of work represented by the board are:
+
+- {guilabel} `Backlog` represents items that haven't been started and not in Sprint.
```

Comment:
![image](https://github.com/2i2c-org/team-compass/assets/3837114/11bebfe6-3b00-4b56-bbef-c77ae6122baf)

I'm new to the `guilabel` thing so I'm not sure on the expected result, but I think this could be a syntax error of having a space between `{guilabel}` and the text in backticks following it.

Reply:
Cool. Thanks!